spec/controllers/review_controller_spec.rb

```diff
@@ -88,231 +88,187 @@
+          post :revert_to_draft, params: { step_by_step_page_id: step_by_step_page.id }
 
-        expect(response).to redirect_to step_by_step_page_path(step_by_step_page)
-      end
+          expect(response).to redirect_to step_by_step_page_path(step_by_step_page)
+        end
 
-      (required_permissions - %w(signin)).each do |required_permission|
-        it "cannot be accessed by users without the #{required_permission} permission" do
-          stub_user.permissions = required_permissions
-          stub_user.permissions.delete(required_permission)
-          post :claim_2i_review, params: { step_by_step_page_id: step_by_step_page.id }
+        (required_permissions - %w(signin)).each do |required_permission|
+          it "cannot be accessed by users without the #{required_permission} permission" do
+            stub_user.permissions = required_permissions
+            stub_user.permissions.delete(required_permission)
+            post :claim_2i_review, params: { step_by_step_page_id: step_by_step_page.id }
 
-          expect(response.status).to eq(403)
+            expect(response.status).to eq(403)
+          end
         end
-      end
 
-      it "sets status to draft and removes the reviewer id and review requester id" do
-        stub_user.permissions = required_permissions
-        post :revert_to_draft, params: { step_by_step_page_id: step_by_step_page.id }
+        it "sets status to draft and removes the reviewer id and review requester id" do
+          stub_user.permissions = required_permissions
+          post :revert_to_draft, params: { step_by_step_page_id: step_by_step_page.id }
 
-        step_by_step_page.reload
+          step_by_step_page.reload
 
-        expect(step_by_step_page.status).to eq("draft")
-        expect(step_by_step_page.reviewer_id).to be_nil
-        expect(step_by_step_page.review_requester_id).to be_nil
-      end
+          expect(step_by_step_page.status).to eq("draft")
+          expect(step_by_step_page.reviewer_id).to be_nil
+          expect(step_by_step_page.review_requester_id).to be_nil
+        end
 
-      it "creates an internal change note" do
-        stub_user.permissions = required_permissions
-        stub_user.name = "Firstname Lastname"
+        it "creates an internal change note" do
+          stub_user.permissions = required_permissions
+          stub_user.name = "Firstname Lastname"
 
-        expected_change_note = "Reverted to draft"
+          expected_change_note = "Reverted to draft"
 
-        post :revert_to_draft, params: { step_by_step_page_id: step_by_step_page.id }
-        step_by_step_page.reload
+          post :revert_to_draft, params: { step_by_step_page_id: step_by_step_page.id }
+          step_by_step_page.reload
 
-        expect(step_by_step_page.internal_change_notes.first.headline).to eq expected_change_note
+          expect(step_by_step_page.internal_change_notes.first.headline).to eq expected_change_note
+        end
```

Comment:
Probably worth viewing this commit with `?w=1` at the end (https://github.com/alphagov/collections-publisher/pull/780/commits/d11716fad9dc264306011a60ee704266f0949f6a?w=1), so that whitespace diffs are ignored.

There were a number of tests which set up the step by step to get it into an "in_review" state, which became noisy and difficult to read. Refactored so that the setup only happens once and all of the tests are now in a "this is already in_review" context.

Reply:
Have spoken in IRL and you've confirmed this makes sense now. We've agreed this commit leaves the code in a better place than it was in, and that the file in its entirety could do with a refactor in a future PR.